api/v1beta1/types.go

```diff
@@ -75,6 +75,24 @@ func (f *ImageFilter) IsZero() bool {
 	return f.Name == nil && len(f.Tags) == 0
 }
 
+// Validate performs validation on [ImageParam], returning a list of field errors using the provided base path.
+// It is intended to be used in the validation webhooks of resources containing [Image].
+func (i *ImageParam) Validate(base field.Path) field.ErrorList {
+	var errors field.ErrorList
+	// Not possible to validate the image if it is missing
+	if i == nil {
+		errors = append(errors, field.Required(&base, "image is required"))
+		return errors
+	}
+	if i.Filter.Name == nil || i.Filter.Tags == nil {
+		errors = append(errors, field.Required(base.Child("Image filter"), "either name or tags of image are missing"))
+	}
+	if i.ImageRef.Name == "" {
+		errors = append(errors, field.Required(base.Child("ORC mage Regerecne"), "Orc image is missing"))
```

Comment:
I first wondered what language that was, but then realized you probably meant:
```suggestion
		errors = append(errors, field.Required(base.Child("ORC Image Reference"), "ORC image is missing"))
```

😅